src/core/book-keeper.js

```diff
@@ -1,9 +1,71 @@
 'use strict';
 
+var util = require('../util');
+var moment = require('moment');
+var eventObj = require('./event/event');
+
+
 class BookKeeper {
-    constructor () {
+    constructor(eventBus) {
+        var self = this;
+        self._eventBus = eventBus;
+
+        /**
+         * Listener for Clearing past records
+         * @param param.type = event type to cleanup
+         * @param param.time = delete events older than this many days
+         */
+        self._eventBus.addEventListener(eventObj['CLEAR_PAST_RECORDS'], this, function(param) {
```

Comment:
While correct, it looks better to use [] notation only when you need it (e.g. when the key is stored in a variable). So change this and the other uses of eventObj to eventObj.CLEAR_PAST_RECORDS and such